.github/ISSUE_TEMPLATE/icon-request.md

```diff
@@ -0,0 +1,25 @@
+---
+name: 🎨 Icon Design Request
+about: Icon design request for a Visual Designer
+title: '{icon metaphor} icon request'
+labels: 'visual design,triage,icon request'
```

Comment:
Milan and I did a purge of labels a little while ago so that we only have labels with a clear purpose / workflow. Do you have a workflow in mind that needs to distinguish icon requests from regular visual design requests? If not, can we avoid creating a new label?
```suggestion
labels: 'visual design,triage'
```

Reply:
I'd like to keep it - icon requests tend to need a short turnaround time, so I'd like to be able to quickly filter specifically on the icon requests to make sure they're being handled.